scripts/collections.cdc

```diff
@@ -143,6 +144,25 @@ pub fun main(address: Address) : {String : MetadataCollection}? {
 		results["RareRooms"] = MetadataCollection(type: Type<@RareRooms_NFT.Collection>().identifier, items: items)
 	}
 
+
+	let motoGPCollection = account.getCapability<&MotoGPCard.Collection{MotoGPCard.ICardCollectionPublic}>(/public/motogpCardCollection)
+	if motoGPCollection.check() {
+		let motoGPNfts = motoGPCollection.borrow()!.getIDs()
+		let items: [MetadataCollectionItem] = []
+		for id in motoGPNfts {
+			let nft = motoGPCollection.borrow()!.borrowCard(id: id)!
+			let metadata = nft.getCardMetadata()!
+			items.append(MetadataCollectionItem(
+				id: id,
+				name: metadata.name,
+				image: metadata.imageUrl,
+				url: "https://motogp-ignition.com/"
```

Comment:
From what I could gather, we could do this as well for a detail view of the NFT if that's what you want for `.find`
```suggestion
				url: "https://motogp-ignition.com/nft/card/".concat(id.toString()).concat("?owner=").concat(address.toString()),
```